flows/utils.py

```diff
@@ -199,11 +200,13 @@ def raw_data_flow(slug, fetch_data_task, validate_data_task, historize=True):
 
 def run(flow):
     """Runs a flow and log errors"""
-    state = flow(return_state=True)
-    maybe_result = state.result(raise_on_failure=False)
-    if isinstance(maybe_result, ValueError):
+    cwd = os.path.dirname(os.path.abspath(__file__))
+    proc = subprocess.Popen(["python", f"{flow}.py"], shell=False, cwd=cwd)
```

Comment:
don't love this... it's fine for now but we should be able to limit memory consumption without this hacky workaround...